### PR TITLE
Update characterswitch.js

### PR DESCRIPTION
**Pull request by Safe870**
MORE LANGUAGES.

### CODE REVIEW

**Comment by zbot473** [comment]
Lgtm

File: loc/ui/characterswitch.js [hunk 1]
```diff
@@ -8,6 +8,12 @@ export default {
     de: 'Abmeldung',
     fr: 'Déconnexion',
     ko: '로그 아웃'
+    bl: 'Излез от профила си'
```
Comment (zbot473) [inline loc/ui/characterswitch.js]: Commas please

File: loc/ui/characterswitch.js [hunk 2]
```diff
@@ -18,5 +24,11 @@ export default {
     de: 'Startseite',
     fr: 'Accueil',
     ko: '홈페이지'
+    bl: 'Начална страница'
```
Comment (zbot473) [inline loc/ui/characterswitch.js]: Here too

**Comment by zbot473** [comment]
Fixed

**Comment by dekdevy** [comment]
👌

**Pull request merged**